events/nickname_events/wc_nickname_events.txt

```diff
@@ -0,0 +1,156 @@
+
+### the Old Murkeye
+WCNIC.0001 = { # Murloc nickname for when the character rule long
+    type = character_event
+	title = WCNIC.0001.t
+	desc = WCNIC.0001.desc
+    theme = default
+	left_portrait = root
+
+    trigger = {
+		NOR = {
+			has_character_flag = had_nickname_event
+            has_character_flag = denied_nickname_event_old_murkeye
+		}
+        AND = {
+            primary_title = { recent_history = { years = 10 } }
+            has_culture_group = culture_group:murloc_group
+            age >= 45
+        }
+	}
+
+    weight_multiplier = {
+		base = 5
+		modifier = {
+			factor = 2.0
+            age >= 55
+		}
+        modifier = {
+			factor = 10.0
+			age >= 65
+		}
+        modifier = {
+			factor = 1.5
+			primary_title = { recent_history = { years >= 20 } }
+		}
+        modifier = {
+			factor = 1.5
+			primary_title = { recent_history = { years >= 30 } }
+		}
+	}
+
+    immediate = {
+		add_character_flag = {
+			flag = had_nickname_event
+		}
+	}
+
+    option = { # the Old Murkeye
+		name = WCNIC.0001.a
+        flavor = {
+			desc = WCNIC.0001.tt.a
+		}
+		
+		set_nickname_effect = { NICKNAME = nick_the_old_murkeye }
+		give_self_nickname_effect = yes
+        add_prestige = 25
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+
+    option = { # None
+		name = WCNIC.0001.b
+		flavor = {
+			desc = WCNIC.0001.tt.b
+		}
+		
+		remove_character_flag = had_nickname_event
+        add_character_flag = {
+			flag = denied_nickname_event_old_murkeye
+		}
+		add_trait = humble
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+}
+
+### The Old
+WCNIC.0002 = { # Human nickname for when the character rule long
+    type = character_event
+	title = WCNIC.0002.t
+	desc = WCNIC.0002.desc
+    theme = default
+	override_background = {
+		event_background = feast 
+	}
+	left_portrait = root
+
+    trigger = {
+		NOR = {
+			has_character_flag = had_nickname_event
+            has_character_flag = denied_nickname_event_the_old
+		}
+        AND = {
+            primary_title = { recent_history = { years = 20 } }
+            has_culture_group = culture_group:human_group
+            age >= 65
+        }
+	}
+
+    weight_multiplier = {
+		base = 5
+        modifier = {
+			factor = 10.0
+			age >= 75
+		}
+        modifier = {
+			factor = 1.5
+			primary_title = { recent_history = { years >= 30 } }
+		}
+        modifier = {
+			factor = 1.5
+			primary_title = { recent_history = { years >= 40 } }
+		}
+	}
+
+    immediate = {
+		add_character_flag = {
+			flag = had_nickname_event
+		}
+	}
+
+    option = { # the Old
+		name = WCNIC.0002.a
+		
+		set_nickname_effect = { NICKNAME = nick_the_old }
+		give_self_nickname_effect = yes
+		add_prestige = 25
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+
+    option = { # None
+		name = WCNIC.0002.b
+		
+		remove_character_flag = had_nickname_event
+        add_character_flag = {
+			flag = denied_nickname_event_the_old
+		}
+		add_trait = humble
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+}
```

Comment:
These two events can be united into one with different localization.

Reply:
done in b2a25105